CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 1.0.0-nullsafety
```

Comment:
```suggestion
## 1.0.0-nullsafety.0
```

Reply:
Lol, I violated the rule I said earlier this week we should always follow :-)